include/zmq.h

```diff
@@ -399,6 +400,10 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg_,
 /*  RADIO-DISH protocol                                                       */
 #define ZMQ_GROUP_MAX_LENGTH 255
 
+/*  ZMQ_RECONNECT_STOP options                                                */
+#define ZMQ_RECONNECT_STOP_CONN_REFUSED 0x1
+#define ZMQ_RECONNECT_STOP_HANDSHAKE_FAILED 0x2
```

Comment:
this is not used anywhere?

Reply:
If you're referring to `ZMQ_RECONNECT_STOP_HANDSHAKE_FAILED`, that's correct.  The reasons it's included here are:
- To make clear that there are multiple conditions which should stop reconnecting, and that those will be treated as bit-flags in options.
- To support further changes which address failed handshake, but those are a WIP and are not ready yet.

It seemed better to start with something small which is known to work before extending the functionality.  The `ZMQ_RECONNECT_STOP_HANDSHAKE_FAILED` can come out if that would be better.

Reply:
I think it would be confusing for users if there is an option that is not yet implemented but can be set and does nothing. I'd prefer to leave it out, and add it when the implementation is added.

Reply:
Makes sense -- just want to make sure that defining options as a bit-wise OR is acceptable, as I didn't see that approach used elsewhere.